modules/agar/src/main/ts/ephox/agar/assertions/ApproxStructures.ts

```diff
@@ -1,4 +1,4 @@
-import { Assert, assert, TestLabel } from '@ephox/bedrock-client';
+import { Assert, TestLabel } from '@ephox/bedrock-client';
```

Comment:
This intentionally uses `Assert` from bedrock instead of `assert` from chai so that we don't have to add chai as a dependency to agar.